src/components/Hero/index.tsx

```diff
@@ -16,7 +16,7 @@ export const Hero = () => {
         <p className="hero__subtitle">A messaging bot for monitoring and debugging Kubernetes clusters.</p>
         <div className={styles.mainSloganLinks}>
           <Link
-            to={useBaseUrl("/docs/installation")}
+            to={useBaseUrl("/docs/installation/")}
```

Comment:
This is fine either with or without "/" Docusaurus handles it correctly. The problem is in IntegrationList component, replace the code with this.
`src/components/IntegrationList/index.tsx`

```
export const IntegrationList = (props: Props) => {
  const { integrations = [] } = props;
  const currentPath = window.location.pathname.replace(/\/$|$/, "/");

  return (
    <div className={styles.integrations}>
      {integrations.map(integration => (
        <div key={integration.name} className={styles.integrationsItem}>
          <Link to={`${currentPath}${integration.link}`} className="integrations__link">
            <img src={integration.image} alt={integration.name} className="integrations__image" />
            {integration.name}
          </Link>
        </div>
      ))}
    </div>
  );
};
```

Reply:
Ok, I tried to use that, however the CI fails with such error: https://github.com/kubeshop/botkube-docs/actions/runs/3127760256/jobs/5074760131

When I revert that, CI is green.